pkg/ccloudv2/srcm.go

```diff
@@ -4,12 +4,13 @@ import (
 	"context"
 	"net/http"
 
+	srcmv3 "github.com/confluentinc/ccloud-sdk-go-v2-internal/srcmv3/v3"
 	srcmv2 "github.com/confluentinc/ccloud-sdk-go-v2/srcm/v2"
 
 	"github.com/confluentinc/cli/v3/pkg/errors"
 )
 
-func newSrcmClient(httpClient *http.Client, url, userAgent string, unsafeTrace bool) *srcmv2.APIClient {
+func newSrcmV2Client(httpClient *http.Client, url, userAgent string, unsafeTrace bool) *srcmv2.APIClient {
```

Comment:
Why are we still using V2? Hasn't everything switched over to V3?

Reply:
Not yet, srcm/v2 has not been deprecated yet

Reply:
Got it, hopefully the plan is to include all of the V2 endpoints used in the CLI in V3!

Reply:
just realized that srcm/v2 will be deprecated when srcm/v3 is released (by definition) 🤦‍♂️